src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -541,7 +540,7 @@ transactionPreProcessImpl(
         return err;
     }
 
-    std::shared_ptr<STTx> stpTrans;
+    std::shared_ptr<STTx> stTx;
```

Comment:
Please avoid Hungarian notation

Reply:
This isn't hungarian notation, just `STTx` in `camelCase`.